ports/servoshell/webview.rs

```diff
@@ -443,7 +449,8 @@ where
             }
             match msg {
                 EmbedderMsg::Status(_status) => {
-                    // FIXME: surface this status string in the UI somehow
+                    self.hovered_link = _status;
```

Comment:
In rust variables prefixed with `_` are meant to be unused so you should rename `_status` to `status`.

Reply:
I didn't notice that, thanks for pointing it out. have updated now